object-fit-gallery-fetch/main.js

```diff
@@ -1,61 +1,37 @@
-var thumbs = document.querySelectorAll('.thumb');
-var mainImg = document.querySelector('.main');
+const thumbs = document.querySelectorAll(".thumb");
+const mainImg = document.querySelector(".main");
 
-for(var i = 1; i <= thumbs.length ; i++) {
-  var requestObj = 'images/pic' + i + '.jpg';
-  retrieveImage(requestObj, i - 1);
-}
+thumbs.forEach((thumb, index) => {
+  const requestObj = `images/pic${index}.jpg`;
```

Comment:
Should it be `` `images/pic${index + 1}.jpg` `` since the index before started at 1?